internal/mode/static/manager.go

```diff
@@ -107,7 +107,8 @@ func StartManager(cfg config.Config) error {
 		Namespace: cfg.GatewayPodConfig.Namespace,
 		Name:      cfg.ConfigName,
 	}
-	if err := registerControllers(ctx, cfg, mgr, recorder, logLevelSetter, eventCh, controlConfigNSName); err != nil {
```

Comment:
What did the linter complain about here?

Reply:
it was shadowing the declaration of `err`. I ended up not enabling that linter for now because there a few of these and one that I couldn't fix.